stm32cube/stm32f3xx/README

```diff
@@ -36,14 +36,20 @@ License Link:
 
 Patch List:
 
-   *Changes from official delivery:
-    -dos2unix applied
-    -trailing white spaces removed
-
-    *Fix warnings for extraneous parentheses
-      Using clang 7.0.1, if ((htim->State == HAL_TIM_STATE_BUSY))
-      generates warnings.  Remove the extra parentheses
-     Impacted files:
-      drivers/src/stm32f3xx_hal_tim.c
-      drivers/src/stm32f3xx_hal_tim_ex.c
-     ST Bug tracker ID: 63616
```

Comment:
If bug fix is not present anymore (because corrected in official cube package), let's state it in commit message

Reply:
Patch list is now the result of the '_git commit -as_' after applying the changes from the stm32Cube  newest version.
- [master 21aaa39] new_version
 178 files changed, 144242 insertions(+), 129526 deletions(-)
 rename stm32cube/stm32f3xx/drivers/include/{stm32f3xx_hal_conf.h => stm32f3xx_hal_conf_template.h} (73%)
 create mode 100644 stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_exti.h
 rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_pcd.h (66%)
 rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_pcd_ex.h (74%)
 rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_smartcard_ex.h (77%)
 rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_tim_ex.h (85%)
 rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_usart_ex.h (66%)
 create mode 100644 stm32cube/stm32f3xx/drivers/include/stm32f3xx_ll_usb.h
 create mode 100644 stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_exti.c
 rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_i2s_ex.c (78%)
 rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_nand.c (64%)
 rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_opamp_ex.c (75%)
 rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_pcd_ex.c (72%)
 create mode 100644 stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_usart_ex.c
 create mode 100644 stm32cube/stm32f3xx/drivers/src/stm32f3xx_ll_usb.c